pkg/cue/process/handle.go

```diff
@@ -134,7 +139,18 @@ func NewContextWithHooks(namespace, name, appName, appRevision string, baseHooks
 	}
 }
 
-// SetBase set templateContext base model
+// NewPolicyContext create Application Scope templateContext for Policy
+func NewPolicyContext(namespace, name, appName, appRevision string, components []common.ApplicationComponent) Context {
+	return &templateContext{
+		name:        name,
```

Comment:
name is the component name, which should not be in context of Policy

Reply:
this name represent the name of policy

Reply:
ok, that's fine